pipelines/whale/extractor/bigquery_metadata_extractor.py

```diff
@@ -79,9 +100,15 @@ def _retrieve_tables(self, dataset) -> Any:
                             total_cols = 0
                             for column in schema["fields"]:
                                 total_cols = self._iterate_over_cols(
-                                    "", column, cols, total_cols + 1
+                                    tags_dict, "", column, cols, total_cols + 1
                                 )
 
+                table_tag = None
+                if "tags" in tags_dict:
+                    for tag in tags_dict["tags"]:
+                        if not "column" in tag:
+                            table_tag = tag
```

Comment:
Huh. I didn't know you could do `not x in y`. I always write it `x not in y`. Learning something new every day :)

Reply:
I could change it to make it clearer for everybody :)